### PR TITLE
Add missing if block to VR tutorial source example

### PR DESCRIPTION
**Pull request by xyzzy45**
Fixed formatting and corrected the example code to better reflect the intent of the tutorial text.
i.e.  "...The last thing we are doing in button_pressed is checking to see if the button pressed is 1..."

### CODE REVIEW

**Comment by BastiaanOlij** [comment]
CC @TwistedTwigleg 

I have an older version of the demo here which doesn't have the menu button yet but code change does make sense as best as I can tell

**Comment by TwistedTwigleg** [comment]
Yeah, I think this change makes sense.

I wonder if the included zip file also has needs this change as well. My hunch is that it does not, because looking at the code on the tutorial right now I am fairly positive it does not work. My hunch is that when I copied-pasted the additional functionality (like the menu button and trackpads), I forgot a line or two.
Though If we need to, we can always change the zip file later.

Thanks for spotting and making a PR fixing this @xyzzy45!
(Thanks for the ping @BastiaanOlij )

File: tutorials/vr/vr_starter_tutorial.rst [hunk 1]
```diff
@@ -377,7 +377,8 @@ Add the following to ``VR_Controller.gd``:
         
         
         # If the menu button is pressed...
-        if grab_mode == "AREA":
+	if button_index == 1:
```
[the remark this answers is not in the document]
Comment (xyzzy45) [reply tutorials/vr/vr_starter_tutorial.rst]: all set

**Comment by mhilbrunner** [comment]
Thanks for this @xyzzy45 and thanks for keeping up the support of the tutorials, @TwistedTwigleg :)

**Pull request merged**